scripts/run_travis_tests.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-git clone --depth=1 https://github.com/jgraph/mxgraph.git
+git clone --depth=1 --branch v3.7.5 https://github.com/jgraph/mxgraph.git
```

Comment:
Please just add a comment that we don't support the latest version at this time (perhaps also a note on README)